### PR TITLE
fix: modify generated from schedule's backup name timestamp to UTC ti…

### LINKED ISSUE #4279: `velero backup create --from-schedule` labels backup in local time
**What steps did you take:**

```bash
velero create schedule xxx '--schedule=@every 15m' --ttl 1h
# a couple hours later…
velero backup create --from-schedule xxx
velero get backup
```

**What did you expect to happen:**

To see a list of backups with the newly manually scheduled backup at the top, all with `name`s based on UTC time of backup creation.

**What happened instead:**

```
NAME                 STATUS            ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
xxx-20211025151320   PartiallyFailed   12       51         2021-10-25 11:13:20 -0400 EDT   48m       default            <none>
xxx-20211025145809   PartiallyFailed   34       0          2021-10-25 10:58:09 -0400 EDT   33m       default            <none>
xxx-20211025144309   PartiallyFailed   31       0          2021-10-25 10:43:09 -0400 EDT   18m       default            <none>
xxx-20211025142809   PartiallyFailed   21       0          2021-10-25 10:28:09 -0400 EDT   3m        default            <none>
xxx-20211025141309   Deleting          25       0          2021-10-25 10:13:09 -0400 EDT   11m ago   default            <none>
xxx-20211025112340   InProgress        0        0          2021-10-25 11:23:40 -0400 EDT   58m       default            <none>
```

You can see that the in-progress backup I just made is given a `name` based on the local time on my computer, whereas the regular backups made by the server process are labeled in UTC. Thus the manually created backup is displayed in an unnatural sort order.

**Environment:**

Client v1.7.0; server locally patched but recent (from #4242). EKS v1.21.2.

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

### PR DESCRIPTION
**Pull request by blackpiglet**
…mezone.

When backup is created from schedule, and the backup name is not specified, a containing-timestamp generated name will be used. Due to velero client not set timezone to UTC, a local timezone will be used for the generated name.

Singed-off-by: Xun Jiang <jxun@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #4279

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/cmd/cli/backup/create.go [hunk 1]
```diff
@@ -290,7 +290,8 @@ func (o *CreateOptions) BuildBackup(namespace string) (*velerov1api.Backup, erro
 			return nil, err
 		}
 		if o.Name == "" {
-			o.Name = schedule.TimestampedName(time.Now())
+			o.Name = schedule.TimestampedName(time.Now().UTC())
+			//o.Name = schedule.TimestampedName(time.Now())
```
Comment (jenting) [inline pkg/cmd/cli/backup/create.go:294]: Could you please review this line?

**Review by jenting — CHANGES_REQUESTED** [review]

File: pkg/cmd/cli/backup/create.go (hunk 1 above)
Comment (blackpiglet) [reply pkg/cmd/cli/backup/create.go]: Thanks. This line should be removed, but I'm not sure why the documents update is also included in the updated PR.
It seems it's better to open a new PR.

Comment (jenting) [reply pkg/cmd/cli/backup/create.go]: Yes, please check why the documents update is included in this PR.

Comment (blackpiglet) [reply pkg/cmd/cli/backup/create.go]: OK. I will close this PR, and continue on the new one.

**Pull request closed without merging**